src/app/core/common-components/entity-select/entity-select.component.html

```diff
@@ -43,12 +43,29 @@
     (optionSelected)="selectEntity($event.option.value)"
     autoActiveFirstOption
   >
-    <ng-content select="mat-option"></ng-content>
     <mat-option *ngFor="let res of filteredEntities" [value]="res">
       <app-display-entity
         [entityToDisplay]="res"
         [linkDisabled]="true"
       ></app-display-entity>
     </mat-option>
+    <mat-option
+      *ngIf="
+        filteredEntities.length < 1 &&
+        !includeInactive &&
+        inactiveFilteredEntities.length > 0
+      "
+      [disabled]="true"
+      style="display: none"
+    ></mat-option>
```

Comment:
This doesn't seem to ever be displayed?

Reply:
True, but it has to be there in order for the mat-checkbox to be displayed. I've added a comment for that in the code.